src/Test/L0/Worker/StepsRunnerL0.cs

```diff
@@ -590,7 +590,7 @@ private Mock<IActionRunner> CreateStep(TestHostContext hc, TaskResult result, st
             step.Setup(x => x.Condition).Returns(condition);
             step.Setup(x => x.ContinueOnError).Returns(new BooleanToken(null, null, null, continueOnError));
             step.Setup(x => x.Action)
-                .Returns(new DistributedTask.Pipelines.ActionStep()
+                .Returns(new GitHub.DistributedTask.Pipelines.ActionStep()
```

Comment:
why do we need to change this?

Reply:
The compiler was complaining about namespace ambiguity.

Without the qualification, it assumes that `DistributedTask` refers to new namespace `GitHub.Runner.Common.Tests.DistributedTask` (introduced in new test file, `TimelineRecordL0.cs`)